### ISSUE TITLE
DB/Quest:  Replenishing the Healing Crystals

### ISSUE THREAD

**Issue by jududdar** [TrinityCore/TrinityCore#15343]
3.3.5 Branch
hash 603f49a55134
The quest "Replenishing the Healing Crystals" http://www.wowhead.com/quest=9369/replenishing-the-healing-crystals is not given.

There are two versions of this quest, one for Draenei (quest ID 9280) and one for other alliance races (quest ID 9369).  The Draenei only quest is given to Draenei, but the other race one is not - sampled on Human Paladin and Night Elf Druid.  From what little I know about querying the database to find information, it appears that this quest is requiring quest ID 9279.  This is correct if you're a Draenei, but other races cannot get the "You Survived!" quest.

**Comment by Killyana** [comment]
Try:
``` sql
UPDATE `quest_template_addon` SET `PrevQuestID`=0 WHERE `id`=9369;

**Comment by tkrokli** [comment]
Is there any useful info in `quest_template` related to this?
```sql
SELECT `ID`,`Method`,`QuestLevel`,`MinLevel`,`QuestSortID`,`RequiredRaces`,`NextQuestIdChain`,`RewardXPId`,`RewardOrRequiredMoney`,`RewardMoneyMaxLevel`,`SourceItemId`,`Flags`,`RewardChoiceItemID1`,`RewardChoiceItemID2`,`RewardChoiceItemID3`,`RewardChoiceItemQuantity1`,`RewardChoiceItemQuantity2`,`RewardChoiceItemQuantity3`,`RewardFactionID1`,`LogTitle`,`LogDescription`,`QuestDescription`,`EndText`,`OfferRewardText`,`RequestItemsText`,`QuestCompletionLog`,`RequiredNpcOrGo1`,`RequiredItemId1`,`RequiredItemCount1`,`VerifiedBuild` FROM `quest_template` WHERE `id` IN (9280,9369,9409);
```
```
    ID  Method  QuestLevel  MinLevel  QuestSortID  RequiredRaces  NextQuestIdChain  RewardXPId  RewardOrRequiredMoney  RewardMoneyMaxLevel  SourceItemId   Flags  RewardChoiceItemID1  RewardChoiceItemID2  RewardChoiceItemID3  RewardChoiceItemQuantity1  RewardChoiceItemQuantity2  RewardChoiceItemQuantity3  RewardFactionID1  LogTitle                           LogDescription                                                                                                         QuestDescription                                                                                                                                                                                                                                                                                                                                                                                  EndText  OfferRewardText                                                                                                                                                                                                                RequestItemsText                                                                                                                                    QuestCompletionLog                                    RequiredNpcOrGo1  RequiredItemId1  RequiredItemCount1  VerifiedBuild
------  ------  ----------  --------  -----------  -------------  ----------------  ----------  ---------------------  -------------------  ------------  ------  -------------------  -------------------  -------------------  -------------------------  -------------------------  -------------------------  ----------------  ---------------------------------  ---------------------------------------------------------------------------------------------------------------------  ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------  -------  -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------  --------------------------------------------------------------------------------------------------------------------------------------------------  ----------------------------------------------------  ----------------  ---------------  ------------------  -------------
  9279       2           1         1         3526           1024              9280           3                      0                   30             0  524424                    0                    0                    0                          0                          0                          0               930  You Survived!                      Speak with Proenitus at the Crash Site in Ammen Vale.                                                                  Nevermind my injuries, $g brother : sister;. It's a miracle that any of us are alive!$b$bIt didn't look like you'd pull through; you've been in suspension for weeks.$B$BSomething happened and the Exodar crashed. Many have died.$B$BBut I'm relieved that you are alive, $N. With you leading our efforts, I'm sure we'll get to safety soon! Proenitus asked me to send you to him when you awoke. He's$N, by my eternal soul, it is good to see you!  I don't know why the Exodar crashed.  Only those of us that were in this part of the ship remain.$B$BThere's not much time if we're to save everyone that we can!                                                                                                                                                                  (NULL)                                                               0                0                   0          12340
  9280       2           1         1         3526           1024                 0           5                     15                   60             0  524424                24129                24131                24130                          1                          1                          1               930  Replenishing the Healing Crystals  Acquire 8 Vials of Moth Blood from the nearby Vale Moths and give them to Proenitus at the Crash Site in Ammen Vale.   Our immediate concern is the depleted state of our healing crystals.  We must replenish them quickly!$B$BUnfortunately, the only way to do this at this point is to extract the blood from the local vale moths.  Their blood will make for a suitable substitute.$B$BThe moths can be found in many places around the vale, but they are mostly concentrated in the northwest.                            Regrettable that those creatures had to die so that we might live - indiscriminate killing is not the draenei way.  However, the blood you gathered will replenish our healing crystals, so their deaths will not be in vain.  $N, have you harvested the necessary moth blood for the healing crystals yet?  With survivors scattered all over the vale, time is of the essence!  Return to Proenitus at the Crash Site in Ammen Vale.                 0            22889                   8          12340
  9369       2           1         1         3526             77              9409           5                     15                   60             0     136                24129                24131                24130                          1                          1                          1               930  Replenishing the Healing Crystals  Acquire 8 Vials of Moth Blood from the nearby Vale Moths and bring them to Proenitus at the Crash Site in Ammen Vale.  We are the draenei.  Our ship crashed and, since we have many wounded, our immediate concern is the depleted state of our healing crystals.  We must replenish them quickly!$B$BUnfortunately, the only way to do this at this point is to extract the blood from the local vale moths.  Their blood will make for a suitable substitute.$B$BThe moths can be found in many places around the vale(NULL)mosRegrettable that those creatures had to die so that we might live - indiscriminate killing is not the draenei way. However, the blood you gathered will replenish our healing crystals, so their deaths will not be in vain.   Have you harvested the necessary moth blood for the healing crystals yet? With survivors scattered all over the vale, time is of the essence!       Return to Proenitus at the Crash Site in Ammen Vale.                 0            22889                   8          12340
  9409       2           2         1         3526              0                 0           2                      0                   30         23568  524416                    0                    0                    0                          0                          0                          0               930  Urgent Delivery!                   Deliver the Bundle of Vials to Zalduun at the Crash Site in Ammen Vale.                                                Now that you've gathered all that we need to replenish our healing crystals, I need you to deliver the vials to Zalduun, our priest.  You'll find him tending to our wounded inside the central chamber of the ship.                                                                                                                                                                                       Good work!  This will do nicely to replenish our depleted healing crystals.$B$BBut now that you're here, I think I have another use for you, if you're willing?                                                                Hello, $c.  What have you there? Is it for me?                                                                                                      (NULL)                                                               0            23568                   1          12340
```
BTW, I think 1101 is missing in `RequiredRaces` for quest 9409, but maybe that is redundant info?
What about  missing 9409 in `NextQuestIdChain` for quest 9280, is that redundant information too?

**Comment by jududdar** [comment]
Ran query against database and restarted server.  Proenitus still doesn't give the quest.  Verified in my DB that 9369 does not have a PrevQuestID anymore.  Cleared my cache just to make sure.

Oddly enough, I logged on my GM account to query my character to make sure I didn't develop amnesia and had already completed the quest - I had not completed it, but the quest is being offered to my GM character.
